github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-periodics.yaml

```diff
@@ -24,21 +24,95 @@ periodics:
-          ssp_it_path="${ssp_dir}/data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml"
-          ssp_pref_path="${ssp_dir}/data/common-instancetypes-bundle/common-clusterpreferences-bundle.yaml"
+          latest_version=$(curl --fail -s "https://api.github.com/repos/kubevirt/common-instancetypes/releases/latest" | jq -r '.tag_name')
+          description="Automated update of common-instancetypes bundles to ${latest_version}\n\n\\\`\\\`\\\`release-note\nUpdated common-instancetypes bundles to ${latest_version}\n\\\`\\\`\\\`"
 
-          latest_version=$(curl -s https://api.github.com/repos/kubevirt/common-instancetypes/releases/latest | jq -r '.tag_name')
-          base_url="https://github.com/kubevirt/common-instancetypes/releases/download"
-          it_url="${base_url}/${latest_version}/common-clusterinstancetypes-bundle-${latest_version}.yaml"
-          pref_url="${base_url}/${latest_version}/common-clusterpreferences-bundle-${latest_version}.yaml"
+          git-pr.sh -c "cd ../kubevirt && hack/bump-common-instancetypes.sh" -d "echo -e \"${description}\"" -r kubevirt -b update-common-instancetypes -T main
+          git-pr.sh -c "cd ../ssp-operator && hack/bump-common-instancetypes.sh" -d "echo -e \"${description}\"" -r ssp-operator -b update-common-instancetypes -T main
+        resources:
+          requests:
+            memory: "200Mi"
+  - name: periodic-update-common-instancetypes-bundles-release-0.4
+    cron: 0 2 * * *
+    annotations:
+      testgrid-create-test-group: "false"
+    decorate: true
+    decoration_config:
+      timeout: 1h
+    max_concurrency: 1
+    cluster: kubevirt-prow-control-plane
+    extra_refs:
+    - org: kubevirt
+      repo: kubevirt
+      base_ref: release-1.2
+    - org: kubevirt
+      repo: ssp-operator
+      base_ref: release-v0.19
+    labels:
+      preset-github-credentials: "true"
+    spec:
+      containers:
+      - image: quay.io/kubevirtci/pr-creator:v20230103-9f4e101
+        env:
+          - name: COMMON_INSTANCETYPES_BRANCH
+            value: "release-0.4"
+          - name: KUBEVIRT_BRANCH
+            value: "release-1.2"
+          - name: SSP_BRANCH
+            value: "release-v0.19"
+        command: [ "/bin/bash", "-ce" ]
+        args:
+        - |
+          export GIT_ASKPASS=/usr/local/bin/git-askpass.sh
 
+          latest_version=$(curl --fail -s https://api.github.com/repos/kubevirt/common-instancetypes/releases | jq -r '.[] | select(.target_commitish == '\""${COMMON_INSTANCETYPES_BRANCH}"\"') | .tag_name' | head -n1)
+          description="Automated update of common-instancetypes bundles to ${latest_version}\n\n\\\`\\\`\\\`release-note\nUpdated common-instancetypes bundles to ${latest_version}\n\\\`\\\`\\\`"
+
+          git-pr.sh -c "cd ../kubevirt && hack/bump-common-instancetypes.sh ${COMMON_INSTANCETYPES_BRANCH}" -d "echo -e \"${description}\"" -r kubevirt -b update-common-instancetypes -T ${KUBEVIRT_BRANCH}
```

Comment:
it might make sense to have unique branch names here as these jobs could end up running at the same time - maybe something like `update-common-instancetypes-1.2`

Same goes for the rest of the branches.